eng/common/pipelines/templates/steps/create-apireview.yml

```diff
@@ -41,7 +41,7 @@ steps:
       inputs:
         filePath: ${{ parameters.SourceRootPath }}/eng/common/scripts/Create-APIReview.ps1
         arguments: >
-          -PackageInfoFiles ('${{ convertToJson(parameters.PackageInfoFiles) }}' | ConvertFrom-Json -NoEnumerate)
+          -PackageInfoFiles @('${{ join(''',''', parameters.PackageInfoFiles) }}')
```

Comment:
Based on https://learn.microsoft.com/en-us/azure/devops/pipelines/process/expressions?view=azure-devops#join wouldn't this just be `join(',', parameters.PackageInfoFiles)`